core/src/main/java/com/expediagroup/streamplatform/streamregistry/StreamRegistryApplication.java

```diff
@@ -246,7 +245,7 @@ public static SchemaManager loadSchemaManager(StreamRegistryConfiguration config
 
         try {
             SchemaManager schemaManager = Utils.newInstance(schemaManagerClass, SchemaManager.class);
-            schemaManagerConfig.getProperties().put(MAX_SCHEMA_VERSIONS_CAPACITY, 100);
+            schemaManagerConfig.getProperties().put(SchemaManager.MAX_SCHEMA_VERSIONS_CAPACITY, 100);
```

Comment:
Intellij did this as part of the package rename. 🤷‍♂ It screwed up a bunch of other random things, like removing package names from fully qualified class names in yaml files ❗️ 

I prefer readability and sometimes the class name gets in the way, like in this example. The only benefit is if you're reading outside the IDE then you can see where the constant comes from.

Anyway, I've put it back the way it was. It'll likely disappear in the long run anyway. `MAX_SCHEMA_VERSIONS_CAPACITY` is only there because Confluent use a `HashMap` of capped size instead of an LRU cache.